documentation/source/user_section/tutorials/4d/processing-fmri-data/template-registration-for-fmri.rst

```diff
@@ -0,0 +1,42 @@
+Registering fMRI data to the PAM50 template
+###########################################
+
+Now that we have the motion-corrected time-averaged fMRI image, we can use it to register the template to the fMRI image space.
+
+.. code::
+
+   sct_register_multimodal -i "${SCT_DIR}/data/PAM50/template/PAM50_t2s.nii.gz" \
+                           -d fmri_moco_mean.nii.gz \
+                           -dseg t2_seg_reg.nii.gz \
+                           -param step=1,type=im,algo=syn,metric=CC,iter=5,slicewise=0 \
+                           -initwarp ../t2s/warp_template2t2s.nii.gz \
+                           -initwarpinv ../t2s/warp_t2s2template.nii.gz \
+                           -owarp warp_template2fmri.nii.gz \
+                           -owarpinv warp_fmri2template.nii.gz \
+                           -qc ~/qc_singleSubj
+
+.. TODO: I don't understand the choices made for ``-dseg`` and ``-iseg``.
+
+   * Why do we supply a segmentation at all if we're using ``type=im``? Won't this ignore the segmentation image?
+   * Why do we supply t2_seg_reg.nii.gz for ``-dseg`` when ``-d`` is fmri_moco_mean? These don't seem to match up, since ``t2_seg_reg.nii.gz`` was registered to the **non-motion-corrected** fMRI image.
+   * Why don't we supply ``-iseg``? Wouldn't "$SCT_DIR/data/PAM50/template/PAM50_cord.nii.gz" be suitable?
+
+:Input arguments:
+   - ``-i`` : Source image. Here, we select the T2* version of the PAM50 template, because the T2* contrast is the closest visual match to our fMRI data.
+   - ``-d`` : Destination image.
+   - ``-param`` : The parameter settings worth noting are:
+      - ``type=im`` : Since fMRI cannot be segmented reliably (due to low contrast between the spinal cord and the surrounding cerebrospinal fluid), we rely on just the anatomical images (``-i`` and ``-d``) rather than segmentation images (``-iseg`` and ``-dseg``).
+      - ``algo=syn`` : This algorithm helps to compensate for the the lack of segmentation during registration. (TODO: The presenter notes describes this as "ANTs Superpower". But, can we give a more specific reason for why this is needed here in particular? My intent is to answer the question "If syn is so good, why don't we use it elsewhere by default? Why is it suited specifically for this situation?)
+      - ``iter=5``: We decrease the number of iterations (default ``10``) because the registration is sensitive to the artifacts (drop out) in the image. (TODO: The presenter notes could be more specific here, too. Why is this registration specifically sensitive to artifacts? Why don't we do this elsewhere? Is it due to fMRI data? Is it because we're not using a segmentation here?)
```

Comment:
`algo=syn` seems to have personalised notes in bracket. Are we keeping it for public?

Reply:
Definitely not! I tried to highlight this in an earlier comment, actually, but got no response hehe. See: https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3493#discussion_r686227865